docker/ranger/Dockerfile

```diff
@@ -1,23 +1,24 @@
 FROM ubuntu:18.04
-MAINTAINER Andrew Nowak <an8@sanger.ac.uk>
+LABEL org.opencontainers.image.authors="Andrew Nowak <an8@sanger.ac.uk>"
 
 # Hold versions in environment variables.
 ENV NODE_VERSION=10.16.3 \
     MONGODB_VERSION=3.6.14 \
     SAMTOOLS_VERSION=1.10+42_g3c4b380 \
-    LIBHTS_PLUGINS_VERSION=201712+irods_4.2.7 \
+    LIBHTS_PLUGINS_VERSION=201712 \
```

Comment:
I have the feeling we need to switch to a more recent version of samtools and then a lot of this may be obsolete.